guide/src/migration.md

```diff
@@ -81,6 +81,61 @@ enum SimpleEnum {
 ```
 </details>
 
+### `PyType::name` now returns `PyResult<String>` instead of `PyResult<Cow<'_, str>>`
+<details open>
+<summary><small>Click to expand</small></summary>
+
+This function previously would try to reuse a raw C API field (`tp_name`) in which case it would return a `Cow::Borrowed`,
+but this caused inconsistency with Python.
+It now always returned a fully owned String.
+
+To migrate, remove any `.into_owned()` and `.into_mut()` calls on the output of `PyType::name`.
+
+Before:
+
+```rust
```

Comment:
```suggestion
```rust,ignore
```

The CI fails to test this example. Because the return type of `name()` changed, this can't compile anymore. It should be fine to just ignore this one.